src/components/TimeElapsed.tsx

```diff
@@ -1,18 +1,17 @@
 import React, { useState } from 'react';
 
-import { formatDistance } from 'date-fns';
+import { formatTimeDifference, useInterval } from '@utils';
 
-import { useInterval } from '@utils';
+interface Props {
+  value: number;
+}
```

Comment:
Since we don't export this and it's only one parameter you could directly type in the function parameters instead of defining an interface

Reply:
It looks a bit cleaner in my opinion to have it as separate interface, but I can revert.